evaluation/fix_spans.py

```diff
@@ -0,0 +1,46 @@
+# Lint as: python3
+"""Automatic best effort span cleaner for SemEval 2021 Toxic Spans."""
+
+import ast
+import csv
+import itertools
+import sys
+
+
```

Comment:
SPECIAL_CHARACTERS = [" ", "\n", "\t"]

Reply:
This could be also {" ", "\n", "\t"} for a more efficient search in line 26.